### DISCUSSION TITLE
I cannot add company's private repo to codemagic

### DISCUSSION THREAD

**Discussion by Lazizbek97** [codemagic-ci-cd/codemagic-docs#923] [Q&A] [answered]
I have created a new account recently. Now, I can add new apps from my own repositories. But even I gave permission for the company's repository, I cannot add new app from there. I meant I cannot find the company's repositories from on repository selection menu. 

How can i add a repository from company's github account?

**Comment by K-Codemagic** [comment, marked as answer]
Hi, to add private repositories, please whitelist the IPs mentioned in [this](https://docs.codemagic.io/getting-started/firewall/) document. 

Secondly, also check [this](https://docs.codemagic.io/getting-started/github/#managing-connected-github-accounts-and-organizations) doc, and if you still face any issues do let me know.

**Reply by K-Codemagic** [reply]
Hi, Any update?

**Reply by K-Codemagic** [reply]
Hi, is it working for you now?

**Comment by Lazizbek97** [comment]
Yes it's working now

**Reply by K-Codemagic** [reply]
Can you please mark the above solution as the answer if it helped you out?
